assets/js/event-providers/contact-form-7.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+document.addEventListener(
+	'wpcf7mailsent',
+	function ( event ) {
+		global.gtag( 'event', 'contact', {
+			// eslint-disable-next-line sitekit/acronym-case
+			event_category: event.detail.contactFormId,
+			event_label: event.detail.unitTag,
+		} );
+	},
+	false
```

Comment:
`false` is the default, so it might be more clear to omit. WDYT?

Reply:
Removed